Open PR instead of issue on document addition (#55)
OpenTermsArchive/contribution-tool: src/translations/default/thanks.mdx

> ## You are now a contributor to the Open Terms Archive project

Reviewer: ```suggestion
## Thank you for your contribution to Open Terms Archive!
```

> To manage the contributions easily, we use GitHub pull requests. So we created one that <a href={url} target="_blank" rel="noopener">you can see here</a>. This allows you to know the status of your contribution and to communicate with us about it.

Reviewer: ```suggestion
Your suggestion has been turned to a <a href={url} target="_blank">pull request</a> on GitHub. You can follow the status of your suggestion there and communicate with us about it by creating an account on GitHub.
```

> We do our best to add your request as fast possible as possible but it may take several days. Indeed, to guarantee a high quality of data we test each contribution to make sure that everything works perfectly.

Reviewer: ```suggestion
In order to guarantee a high quality of data, we test each contribution individually to make sure it works as intended and is stable over time. We do our best to start tracking the document you suggested as fast possible as possible, but these checks may take a few days.
```
